lib/App/index.js

```diff
@@ -143,6 +120,16 @@ class App {
       }
     }
 
+    if (level >= VALIDATION_LEVELS.VERIFIED) {
+      if (appJson.platforms === undefined) {
+        throw new Error('The property `platforms` is required in order to publish a verified app.');
+      }
+
+      if (appJson.support === undefined) {
+        throw new Error('The property `support` is required in order to publish a verified app.');
+      }
```

Comment:
This doesn't verify that `platforms` is an array and `support` a valid support string, not sure if that used to be the case though.

Reply:
The schema takes care of that, the schema just doesn't make sure the property actually exists.